libs/chain-signatures/contract/src/primitives/participants.rs

```diff
@@ -174,6 +186,20 @@ impl Participants {
             self.participants.remove(pos);
         }
     }
+
+    pub fn change_info(
+        &mut self,
+        account_id: AccountId,
+        new_info: ParticipantInfo,
+    ) -> Result<(), Error> {
+        for (a_id, _, info) in self.participants.iter_mut() {
+            if *a_id == account_id {
```

Comment:
Nit: This is slightly easier to read imo.

```suggestion
        for (participant_id, _, info) in self.participants.iter_mut() {
            if *participant_id == account_id {
```